Android/app/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,34 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
```

Comment:
Since this is just a test app, remove all `drawable-*` folders and keep just the main `drawable/` folder.